stable/enterprise/README.md

```diff
@@ -568,213 +669,226 @@ ui:
 
     ```shell
-    export ENGINE_RELEASE=my-engine-release
     export NAMESPACE=anchore
+    export ENGINE_RELEASE=<YOUR_ENGINE_RELEASE_NAME>
+
     kubectl scale deployment --replicas=0 -l app=${ENGINE_RELEASE}-anchore-engine -n ${NAMESPACE}
     ```
-1. **Perform database upgrade**: Upgrade your external database, we suggest you make a backup first. If using a managed cloud database service refer to their documentation.
+
+1. **Perform database backup**: Backup your external database. See the official [PostgreSQL documentation](https://www.postgresql.org/docs/13/backup.html) for guidance. If using a managed cloud database service refer to their documentation.
+
+1. **Perform database upgrade**: Upgrade your external database. See the official [PostgreSQL documentation](https://www.postgresql.org/docs/13/upgrading.html) for guidance. If using a managed cloud database service refer to their documentation.
+
+    > Tip: Leveraging a backup to instantiate a new database instance enables a non-intrusive database upgrade and Enterprise chart migration. This approach preserves the integrity of your original database. By adopting a blue/green deployment strategy for the migration, you gain the advantage of effortless rollbacks in case of migration-related issues.
+
+1. **(Optional) Update Database Hostname**: If you're employing a blue/green deployment strategy for the database upgrade, update the database hostname in your values file and/or existing Kubernetes secrets to point to your newly created database instance. This step is essential for properly configuring the Enterprise chart to use the new database.
 
 1. **Deploy Anchore Enterprise**: Use the converted values file to deploy the new Anchore Enterprise Helm chart.
 
+    >**Note:** If you are **not using existing secrets**, you will need to uncomment the `ADMIN_PASS` and `SET_ADMIN_PASS` export commands below. This is needed to ensure that your Enterprise deployment stores the correct Anchore admin password in the secret.
+
     ```shell
-    export ENTERPRISE_RELEASE=my-enterprise-release
-    export VALUES_FILE_NAME=${PWD}/output/my-values-file.yaml
-    helm install ${ENTERPRISE_RELEASE} -n ${NAMESPACE} -f ${VALUES_FILE_NAME} --set upgradeJob.force=true anchore/enterprise
-    ```
+    export NAMESPACE=anchore
+    export ENTERPRISE_RELEASE=<YOUR_ENTERPRISE_RELEASE_NAME>
+    export ENTERPRISE_VALUES_FILE=${PWD}/output/enterprise.my-values-file.yaml
 
-    > **Note:** The `upgradeJob.force` flag is required to force the upgrade job to run upon installation. This value is not needed for future upgrades. Remember to unset it if passing it in via the command line or helm may persist the value.
+    # If you are not using existing secrets, uncomment the following export commands
+    #
+    # export ADMIN_PASS=$(kubectl get secret -n ${NAMESPACE} ${ENGINE_RELEASE}-anchore-engine-admin-pass -o jsonpath="{.data.ANCHORE_ADMIN_PASSWORD}" | base64 -d -)
```

Comment:
the `base64 -d -` caused me to get an "invalid argument -" on my mac. taking off the `-` gave the decoded pass

Reply:
The `-` is needed for linux but macos doesn't need it. I'm just going to leave it because the error is pretty obvious.